ddprof-lib/src/main/cpp/stackWalker.cpp

```diff
@@ -458,7 +458,7 @@ void StackWalker::checkFault(ProfiledThread* thrd) {
   VMThread *vm_thread = VMThread::current();
   if (vm_thread != NULL && sameStack(vm_thread->exception(), &vm_thread)) {
     if (thrd) {
-      thrd->exitCrashHandler();
+      thrd->resetCrashHandler();
```

Comment:
this is the main change

Reply:
Could you add a comment here about why the reset needs to be done here?